### PR TITLE
Use string concatenation instead of array join for making RPC URLs

### PR DESCRIPTION
**Pull request by httpsOmkar**
From array to string template converted for better readability

### CODE REVIEW

File: packages/firestore/src/platform_browser/webchannel_connection.ts [hunk 1]
```diff
@@ -415,16 +415,6 @@ export class WebChannelConnection implements Connection {
...
-    url.push(this.databaseId.projectId);
-
-    url.push('/databases/');
-    url.push(this.databaseId.database);
-    url.push('/documents');
-
-    url.push(':');
-    url.push(urlRpcName);
-    return url.join('');
+    return `${ this.baseUrl }/${ RPC_URL_VERSION }/projects/${ this.databaseId.projectId }/databases/${ this.databaseId.database }/documents:${ urlRpcName }`;
```
Comment (wilhuff) [inline packages/firestore/src/platform_browser/webchannel_connection.ts]: Is there any way to break this up? 159 characters is almost double the allowed line length.

Comment (httpsOmkar) [reply packages/firestore/src/platform_browser/webchannel_connection.ts]: yeah sure

Comment (httpsOmkar) [reply packages/firestore/src/platform_browser/webchannel_connection.ts]: prettier is reverting my changes

Comment (wilhuff) [reply packages/firestore/src/platform_browser/webchannel_connection.ts]: FWIW, I think the prior arrangement was essentially a workaround for this disconnect.

What about splitting with a `+`?

Comment (httpsOmkar) [reply packages/firestore/src/platform_browser/webchannel_connection.ts]: fixed

**Pull request merged**